code-editors-wiki-article.md

```diff
@@ -0,0 +1,38 @@
+# Code Editors
+
+Source code editors are the programs that allow for code creation and 
+editing. Any text editor can be used to write code. But dedicated code
+editors offer many useful features, including: text completion, 
+markup coloring, and tabs/panes for side-by-side editing. These features can 
+be useful even to new users who don't know their way around all of the 
+advanced options. 
+
+There are many, many editors availble to download or use
+online, both paid and free. They offer many of the same core-features, but
+each have their own subtlties. Further research may help find the one that's
+right for you. The following editors are all cross-platform and 
+free to use or evaluate.
+
+## [Sublime Text](https://www.sublimetext.com/) <img src="https://upload.wikimedia.org/wikipedia/en/4/4c/Sublime_Text_Logo.png" alt="Sublime Text Icon" height=32px />
+![Sublime Text Screenshot](http://cache.filehippo.com/img/ex/2794__Sublime_text_2_1.png)
+
+Sublime Text is a very popular editor that has been around since 2008. There 
+are many options and extensions available, and a lot of help available online 
+to help set it up. (Sublime Text is $70, but generously allows you unlimited 
+use for "evaluation" with a nag-screen pop-up.)
+
+
+## [Brackets](http://brackets.io/) <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Brackets_Icon.svg/1024px-Brackets_Icon.svg.png" alt="Brackets Icon" height=32px />
+![Brackets Screenshot](https://pmoelgaard.files.wordpress.com/2012/06/image-2012-06-26-9-46-57-am.png)
+
+Brackets is a relatively new open-source editor by Adobe. It is very user
+friendly, especially for people who aren't used to command-line interfaces
+or JSON-style settings/prefereces. Extensions and themes are quick and easy 
+to find and install through the Extension Manager.
+
+## [Atom](https://atom.io/) <img src="https://www.pubnub.com/wp-content/uploads/2015/01/B62xj9FCUAA3YoC.png" alt="Atom Icon" height=32px />
+![Atom Screenshot](http://shashikantjagtap.net/wp-content/uploads/2014/03/install-themes.png)
+
+Atom is used as much or more as Sublime Text. It is very configurable 
```

Comment:
> Atom is used as much or more as Sublime Text

I think it would be better if you write it as

> Atom is used as much as Sublime Text, if not more.

The reason is _as_ go very well preceded by _as much_, but _more_ expects a _than_.